build_deploy.sh

```diff
@@ -1,13 +1,15 @@
 #!/bin/bash
 
 TAG=$(git rev-parse --short=7 HEAD)
-
 BM_INVENTORY_IMAGE="quay.io/app-sre/bm-inventory"
+BUILD_FOLDER=$PWD/build
+GIT_REVISION=$(git rev-parse HEAD)
 
-docker build -t "${BM_INVENTORY_IMAGE}:latest" -f Dockerfile.demo .
+skipper make create-build-dir
+CGO_ENABLED=0 go build -o $BUILD_FOLDER/bm-inventory cmd/main.go
```

Comment:
you can also simply call 'skipper make build' instead of lines 8-9

Reply:
Looks like `skipper make build` will also run `lint` and `unit-test`, this is time consuming and unecessary, am I missing something here? @danielerez